cmd/cloud/deployment.go

```diff
@@ -52,6 +54,11 @@ var (
 		# Update a deployment variables from a file
 		$ astro deployment variable update --deployment-id <deployment-id> --load --env .env.my-deployment
 		`
+	wQueueConcurrency    int
+	wQueueMinWorkerCount int
+	wQueueMaxWorkerCount int
+	wQueueWorkerType     string
+	wQueueName           string
```

Comment:
might want to keep these under `deployment_workerqueue.go` where they are actually being used

Reply:
post breaking out `workerqueue` from deployment, I was debating if I should leave the vars in `deployment.go` or move them closer to where they are being used. Thanks for helping me decide :) Closer to where they are used wins!